### DISCUSSION TITLE
Postman environmental variables

### DISCUSSION THREAD

**Discussion by mporzak** [hyperjumptech/monika#700] [Q&A] [answered]
Hi,

When running Postman collection with Monika is it possible to use Postman environmental variables (exported as json file)?

Thanks!

**Comment by ilmiawan** [comment, marked as answer]
HI @mporzak ,

Thanks for asking us this question. Currently Monika haven't been able to read from Postman's environment variables yet.
Could you help to elaborate in what kind of scenarios or use cases you are dealing with for this particular usage ?
